server/app.js

```diff
@@ -15,16 +15,63 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cors());
 
+const calendarModel = require('./src/models/calendar.js');
```

Comment:
this should be split intro controller and route

Reply:
We decided to modularize this post merging to master